cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -0,0 +1,101 @@
+// seed api with sample topology layout
+beforeEach(() => {
+  cy.intercept(
```

Comment:
Curious, is it possible to use real data instead of fixtures here by creating resources on the backend?

Reply:
@vidyanambiar yes we can. I originally went with a fixture because the ec2 test instance only returns one control node for the mesh viz endpoint. Creating an instance via the API is a little more involved than just sending a POST request, so I've documented further details [here](https://github.com/ansible/ansible-ui/pull/1239/commits/38ba46dea893c388f051e74a78cacc88dc0ae74c#diff-bf38aa24220f43988177c19da97801caa738f9a7584447632279c064877a25e7R125). The latest changes should test against the API returned response where possible, and utilize a fixture for the case where we need a specific node to have a specific attribute that renders. Thanks for reviewing!